src/sources/kubernetes_logs/mod.rs

```diff
@@ -219,6 +219,12 @@ pub struct Config {
     #[configurable(metadata(docs::examples = "/path/to/.kube/config"))]
     kube_config_file: Option<PathBuf>,
 
+    /// Replaces `MostRecent` list requests semantic with [resource version][resource_version] `Any`,
+    /// which makes vector using cached resources from kube-apiserver instead of an etcd quorum read.
+    ///
+    /// [resource_version]: https://kubernetes.io/docs/reference/using-api/api-concepts/#semantics-for-get-and-list
```

Comment:
```suggestion
    /// Determines if requests to the kube-apiserver can be served by a cache.
```

I don't love this replacement but I think it's an improvement - I don't know if the enduser needs to background detail. It's probably ok to have a line break and then the lower level details that link to the kube docs. I couldn't write it in a way that didn't feel really awkward though.

Reply:
Ok, no objections. Regenerate docs for a site as well.